src/components/ReservationForm.tsx

```diff
@@ -51,12 +52,25 @@ const ReservationForm: FunctionComponent<CalendarUIProps> = ({
 }) => {
   // Get values from App
   const { user } = useContext(AuthContext);
+
+  useEffect(() => {
+    fetchAllResources(
+      dispatch,
+      CalendarAction.ReceivedAllResources,
+      CalendarAction.Error,
+      `/api/equipment?context=${ResourceKey.Equipment}`
+    );
+  });
```

Comment:
Check if you are calling the API a lot.
```suggestion
  }, []);
```